client/src/groups/GroupApp.jsx

```diff
@@ -0,0 +1,232 @@
+    this.handleCloseUsersDlg = this.handleCloseUsersDlg.bind(this);
+    this.handleCreateGroup = this.handleCreateGroup.bind(this);
+    this.handleAddTag = this.handleAddTag.bind(this);
+    this.handleDeleteTag = this.handleDeleteTag.bind(this);
+    this.handleAddUser = this.handleAddUser.bind(this);
+    this.handleDeleteUser = this.handleDeleteUser.bind(this);
+    this.fetchTagsForGroup = this.fetchTagsForGroup.bind(this);
+    this.fetchUsersForGroup = this.fetchUsersForGroup.bind(this);
+    this.fetchAllTags = this.fetchAllTags.bind(this);
+    this.fetchAllUsers = this.fetchAllUsers.bind(this);
+  }
+
+  async componentDidMount() {
+    this.fetchAllTags();
+    this.fetchAllUsers();
+  }
+
+  handleCreateGroup(name, isUse) {
+    const data = {
+      name,
+      is_use: isUse,
+    };
+
+    axios.post('/super/groups', data, {
+      headers: {
+        Accept: 'application/json',
+      },
+    }).then(() => {
+      window.location.reload();
+    });
+  }
+
+  handleShowTagsDlg(groupId) {
+    this.fetchTagsForGroup(groupId);
+    this.setState({
+      showTagsDlg: true,
+      selectedGroupId: groupId,
+    });
+  }
+
+  handleCloseTagsDlg() {
+    this.setState({
+      showTagsDlg: false,
+      assignedTags: null,
+      selectedGroupId: 0,
+    });
+  }
+
+  handleShowUsersDlg(groupId) {
+    this.fetchUsersForGroup(groupId);
+    this.setState({
+      showUsersDlg: true,
+    });
+  }
+
+  handleCloseUsersDlg() {
+    this.setState({
+      showUsersDlg: false,
+      usersAssigned: null,
+    });
+  }
+
+  handleAddTag(groupId, tagId) {
+    axios.post(`/super/groups/${groupId}/tags`, {
+      tag_id: tagId,
+    }).then(() => {
+      this.fetchTagsForGroup(groupId);
+    }).catch((err) => {
+      alert(err);
+    });
+  }
+
+  handleDeleteTag(groupId, tagId) {
+    axios.delete(`/super/groups/${groupId}/tags/${tagId}`)
+      .then(() => {
+        this.fetchTagsForGroup(groupId);
+      }).catch((err) => {
+        alert(err);
+      });
+  }
+
+  handleAddUser(groupId, userId) {
+    axios.post(`/super/groups/${groupId}/users`, { user_id: userId })
+      .then(() => {
+        this.fetchUsersForGroup(groupId);
+      }).catch((err) => {
+        alert(err);
+      });
+  }
+
+  handleDeleteUser(groupId, userId) {
+    axios.delete(`/super/groups/${groupId}/users/${userId}`)
+      .then(() => {
+        this.fetchUsersForGroup(groupId);
+      }).catch((err) => {
+        alert(err);
+      });
+  }
+
+  fetchAllTags() {
+    axios.get('/super/tags', {
+      headers: {
+        Accept: 'application/json',
+      },
+    }).then((res) => {
+      this.setState(Object.assign({}, this.state, {
+        tags: res.data,
+      }));
+    }).catch((err) => {
+      alert(err);
+    });
+  }
+
+  fetchAllUsers() {
+    axios.get('/super/users', {
+      headers: {
+        Accept: 'application/json',
+      },
+    }).then((res) => {
+      this.setState(Object.assign({}, this.state, {
+        users: res.data.users,
```

Comment:
The result contains only 25 users maybe because of paging. So I cannot find myself in the users dialog even if I'm a member of the group.